hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/common/SparkReaderContextFactory.java

```diff
@@ -86,10 +89,9 @@ class SparkReaderContextFactory implements ReaderContextFactory<InternalRow> {
     Configuration configs = getHadoopConfiguration(jsc.hadoopConfiguration());
     schemaEvolutionConfigs.forEach(configs::set);
     configurationBroadcast = jsc.broadcast(new SerializableConfiguration(configs));
-    // Broadcast: ParquetReader.
-    // Spark parquet reader has to be instantiated on the driver and broadcast to the executors
-    SparkParquetReader parquetFileReader = sparkAdapter.createParquetFileReader(false, sqlConf, options, configs);
-    parquetReaderBroadcast = jsc.broadcast(parquetFileReader);
+    SparkFileReader parquetReader = SparkAdapterSupport$.MODULE$.sparkAdapter().createParquetFileReader(
```

Comment:
I thought we are abstracting these out.
So, why do I still see `createParquetFileReader` 
can you help throw some light